### PR TITLE
Fix/standardize all links, enable docs strict mode

### PR DESCRIPTION
**Pull request by collindutter**
----
📚 Documentation preview 📚: https://griptape--852.org.readthedocs.build//852/

### CODE REVIEW

**Review by emjay07 — APPROVED** [review]

File: docs/griptape-framework/drivers/prompt-drivers.md [hunk 1]
```diff
@@ -356,7 +356,7 @@ The [AmazonSageMakerJumpstartPromptDriver](../../reference/griptape/drivers/prom
 
 Amazon Sagemaker Jumpstart provides a wide range of models with varying capabilities.
 This Driver has been primarily _chat-optimized_ models that have a [Huggingface Chat Template](https://huggingface.co/docs/transformers/en/chat_templating) available.
-If your model does not fit this use-case, we suggest sub-classing [AmazonSageMakerJumpstartPromptDriver](../../reference/griptape/drivers/prompt/amazon_sagemaker_jumpstart_prompt_driver.md) and overriding the [_to_model_input](../../reference/griptape/drivers/prompt/amazon_sagemaker_jumpstart_prompt_driver/#griptape.drivers.prompt.amazon_sagemaker_jumpstart_prompt_driver.AmazonSagemakerJumpstartPromptDriver._to_model_input) and [_to_model_params](../../reference/griptape/drivers/prompt/amazon_sagemaker_jumpstart_prompt_driver/#griptape.drivers.prompt.amazon_sagemaker_jumpstart_prompt_driver.AmazonSagemakerJumpstartPromptDriver._to_model_params) methods.
+If your model does not fit this use-case, we suggest sub-classing [AmazonSageMakerJumpstartPromptDriver](../../reference/griptape/drivers/prompt/amazon_sagemaker_jumpstart_prompt_driver.md) and overriding the `_to_model_input` and `_to_model_params` methods.
```
Comment (dylanholmes) [inline docs/griptape-framework/drivers/prompt-drivers.md:359]: Wanna remove the `.md` here too?

File: docs/griptape-framework/engines/query-engines.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ## Overview
-Query engines are used to search collections of text.
+Query engines are used to perform text queries against various modalities. 
 
-## VectorQueryEngine
+## Vector
 
-Used to query vector storages. You can set a custom [prompt_driver](../../reference/griptape/engines/query/vector_query_engine.md#griptape.engines.query.vector_query_engine.VectorQueryEngine.prompt_driver.md) and [vector_store_driver](../../reference/griptape/engines/query/vector_query_engine.md#griptape.engines.query.vector_query_engine.VectorQueryEngine.vector_store_driver.md). Uses [LocalVectorStoreDriver](../../reference/griptape/drivers/vector/local_vector_store_driver.md) by default.
+Used to query vector storages. You can set a custom [prompt_driver](../../reference/griptape/engines/query/vector_query_engine.md#griptape.engines.query.vector_query_engine.VectorQueryEngine.prompt_driver) and [vector_store_driver](../../reference/griptape/engines/query/vector_query_engine.md#griptape.engines.query.vector_query_engine.VectorQueryEngine.vector_store_driver). Uses [LocalVectorStoreDriver](../../reference/griptape/drivers/vector/local_vector_store_driver.md) by default.
```
Comment (dylanholmes) [inline docs/griptape-framework/engines/query-engines.md:6]: Missed one `.md`

File: docs/griptape-framework/engines/query-engines.md [hunk 3]
```diff
@@ -25,3 +25,28 @@ engine.upsert_text_artifacts(
 
 engine.query("what is griptape?", namespace="griptape")
 ```
+
+## Image
+The [Image Query Engine](../../reference/griptape/engines/image_query/image_query_engine.md) is used to execute natural language queries on the contents of images. You can specify the provider and model used to query the image by providing the Engine with a particular [Image Query Driver](../drivers/image-query-drivers.md).
+
+All Image Query Drivers default to a `max_tokens` of 256. You can tune this value based on your use case and the [Image Query Driver](../drivers/image-query-drivers.md) you are providing.
```
Comment (dylanholmes) [inline docs/griptape-framework/engines/query-engines.md:32]: `.md`

File: docs/griptape-framework/structures/tasks.md [hunk 4]
```diff
@@ -627,7 +627,7 @@ pipeline.run("An image of a mountain shrouded by clouds")
 
 ## Image Query Task
 
-The [Image Query Task](../../reference/griptape/tasks/image_query_task.md) executes a natural language query on one or more input images. This Task uses an [Image Query Engine](../engines/image-query-engines.md) configured with an [Image Query Driver](../drivers/image-query-drivers.md) to perform the query. The functionality provided by this Task depend on the capabilities of the model provided by the Driver.
+The [Image Query Task](../../reference/griptape/tasks/image_query_task.md) executes a natural language query on one or more input images. This Task uses an [Image Query Engine](../engines/query-engines.md#image) configured with an [Image Query Driver](../drivers/image-query-drivers.md) to perform the query. The functionality provided by this Task depend on the capabilities of the model provided by the Driver.
```
Comment (dylanholmes) [inline docs/griptape-framework/structures/tasks.md:630]: `.md`

**Review by dylanholmes — APPROVED** [review]
Love the consistency!

Noticed that you were sometimes removing the `.md` extension from the links and noted some places still had it. I'll leave it to you to decide to change them or not since it doesn't effect functionality.

**Comment by collindutter** [comment]
@dylanholmes the removal of `.md` was only when linking when specific headers on a page. I was resolving via the output of `poetry run mkdocs build` so I feel confident the changes are accurate.

**Pull request merged**